generators/generator-base-private.js

```diff
@@ -759,6 +759,42 @@ module.exports = class extends Generator {
         }
     }
 
+    /**
+     * Instantiates the blueprint generators, if any.
+     * @param {string} subGen - sub generator
+     * @param {any} extraOptions - extra options to pass to blueprint generator
+     * @return {true} useBlueprints - true if one or more blueprints generators have been constructed; false otherwise
+     */
+    instantiateBlueprints(subGen, extraOptions) {
```

Comment:
This should probably belong to `generator-base-blueprint.js`, since only generators that can be overridden by blueprints need this.

Reply:
Fixed